main/src/main/scala/sbt/ScriptedPlugin.scala

```diff
@@ -78,7 +79,14 @@ object ScriptedPlugin extends AutoPlugin {
     scriptedClasspath := getJars(ScriptedConf).value,
     scriptedTests := scriptedTestsTask.value,
     scriptedParallelInstances := 1,
-    scriptedBatchExecution := true,
+    scriptedBatchExecution := {
+      CrossVersionUtil.binarySbtVersion(scriptedSbt.value) match {
+        case "0.13" =>
+          false
+        case _ =>
+          true
+      }
+    },
```

Comment:
```suggestion
    scriptedBatchExecution := CrossVersionUtil.binarySbtVersion(scriptedSbt.value) != "0.13",
```

nit: do we need pattern match?